opyn/opyn/erc20.py

```diff
@@ -56,7 +59,7 @@ def get_allowance(self, owner: str, spender: str) -> int:
 
         response = self.contract.functions.allowance(owner_address, spender_address).call()
 
-        return response
+        return cast(int, response)
```

Comment:
why not `int(response)`?

Reply:
we do not want to introduce an unnecessary cast at runtime, we only want to inform mypy about the type because the contract call is unable to provide a type.
In addition, if `call` will ever be able to provide a type, then cast will become redundant e will be reported as an error so that we will be able to safely removed it. On the other side a int(...) will remain hidden in the code even if no longer needed.

Reply:
Don't think it will remain hidden as an int is very different from a string, but ok.